### PR TITLE
Check if database contains some obsolete stuff

### PR DESCRIPTION
**Pull request by akikoskinen**
Added a couple of system checks. See commit messages for more details.

### CODE REVIEW

File: open_city_profile/checks.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from django.contrib.contenttypes.models import ContentType
+from django.core.checks import register, Tags, Warning
+from django.db import connections
+
+
+@register(Tags.database)
+def check_obsolete_database_tables(app_configs, **kwargs):
+    known_unmanaged_table_names = {"django_migrations"}
```
Comment (tituomin) [inline open_city_profile/checks.py:8]: UPPERCASE var name?

Comment (akikoskinen) [reply open_city_profile/checks.py]: Changed.

**Review by mikkokeskinen — APPROVED** [review]
Code looks ok.

Though, in my opinion these should not be checked every time the server starts but with a management command or similar.

**Comment by akikoskinen** [comment]
> in my opinion these should not be checked every time the server starts but with a management command or similar.

I'm curious: why so? Do you mean these shouldn't be using Django's system check framework? If so, why not?

These checks are tagged with the `database` tag, so they get run only with the `migrate` command or if explicitly requested to run.

**Comment by mikkokeskinen** [comment]
Seems I wasn't familiar enough with the Django's system check framework. This makes my point moot:

> For performance reasons, checks are not run as part of the WSGI stack that is used in deployment. If you need to run system checks on your deployment server, trigger them explicitly using check.

**Comment by akikoskinen** [comment]
> Seems I wasn't familiar enough with the Django's system check framework. This makes my point moot:
> 
> > For performance reasons, checks are not run as part of the WSGI stack that is used in deployment. If you need to run system checks on your deployment server, trigger them explicitly using check.

Yep, that's one point. But another is that the `migrate` command **is** run each time the container starts. So these checks will be run each time too a container starts. But the `migrate` command itself probably causes a lot more database traffic any way, just to figure out what migrations it needs to install, if any.

It's also possible to _not_ run the `migrate` command each time the container starts. That's just how things are configured currently.

**Pull request merged**